amf-api-contract/shared/src/main/scala/amf/apicontract/internal/spec/common/reference/ApiReferenceHandler.scala

```diff
@@ -85,6 +85,7 @@ class ApiReferenceHandler(spec: String) extends ReferenceHandler {
       case Raml10.id | Raml08.id             => ramlLinks(part)
       case Oas20.id | Oas30.id | AwsOas30.id => oasLinks(part)
       case JsonSchema.id                     => oasLinks(part)
+      case AvroSchema.id                     => oasLinks(part) // TODO: what to put here?
```

Comment:
I think, this is for links from the Spec (AVRO in this case) to another. AVRO Schemas cannot have refs, so it should never enter here.

Reply:
great, removed 👌🏽